eshitaban18.md

```diff
@@ -0,0 +1,88 @@
+<a href = 'https://github.com/eshitaban18'> <img width = '32px' align= 'center' src="https://raw.githubusercontent.com/rahulbanerjee26/githubAboutMeGenerator/main/icons/github.svg"/></a>
+
+
+
+
+## Technical Expertise
+
+----
+<div>
+  <p style="display: inline-block;" align="center">
+    <kbd>
+      <kbd>Programming Languages</kbd>
+      <br>
+      <br>
+      <img width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/python/python-original.svg" />
+      <img title="matlab" width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/matlab/matlab-original.svg" />
+      <img width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/r/r-original.svg" /> 
+      <img width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/cplusplus/cplusplus-original.svg" />
+      <img width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/fortran/fortran-original.svg" />
+   </kbd>
+    <kbd>
+      <kbd>Operating systems and Window managers</kbd>
+      <br>
+      <br>
+      <img width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/linux/linux-original.svg" />
+      <img width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/archlinux/archlinux-original.svg" />
+      <img width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/ubuntu/ubuntu-original.svg" />
+      <img width="30px" src="./gifs/bspwm.webp" />
+    </kbd>
+    <kbd>
+      <kbd>Terminal Scripts</kbd>
+      <br>
+      <br>
+      <img width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/bash/bash-original.svg" />
+      <img width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/neovim/neovim-original.svg" />
+      <img width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/vim/vim-original.svg" />
+      </kbd>
+        <kbd>
+      <kbd>Data Analysis</kbd>
+      <br>
+      <br>
+      <img width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/numpy/numpy-original.svg" />
+      <img width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/pandas/pandas-original.svg" />
+      <img width="30px" src="https://scipy.org/images/logo.svg" />
+      <img width="30px" src="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/icons/scikitlearn/scikitlearn-original.svg" />
+    </kbd>
+  </p>
+</div>
+
+## <img src="https://github.com/7oSkaaa/7oSkaaa/blob/main/Images/Statistics.gif?raw=true" width = 50px />  Github Stats
+
+![Top Langs](https://github-readme-stats.vercel.app/api/top-langs/?username=eshitaban18&theme=tokyonight)
+
+Thanks to [this Github repo](https://github.com/durgeshsamariya/awesome-github-profile-readme-templates) for helping me to decorate this ReadMe file.
+
+
+
+
+
+
+
+
+
+
+--------------------------------------------------
+Credit: [username][https://github.com/eshitaban18]
```

Comment:
instead of username add your username

Reply:
I have updated it.

Reply:
Hi, Just one small doubt. I can not see my .md file in templates yet.

Reply:
i deleted that file.
because it was at root level
you need to add that in a template folder
I notice that after merging it so deleted.
If you want create another pull request by adding file in template folder